mailpoet/tests/DataGenerator/Generators/WooCommercePastRevenues.php

```diff
@@ -268,59 +285,66 @@ private function getBatchLog($dataType, $generatedCount) {
+    $connection->executeStatement("ALTER TABLE `{$prefix}postmeta` DISABLE KEYS");
+    $connection->executeStatement("ALTER TABLE `" . $this->entityManager->getClassMetadata(SubscriberEntity::class)->getTableName() . "` DISABLE KEYS");
+    $connection->executeStatement("ALTER TABLE `" . $this->entityManager->getClassMetadata(SubscriberSegmentEntity::class)->getTableName() . "` DISABLE KEYS");
+    $connection->executeStatement("ALTER TABLE `" . $this->entityManager->getClassMetadata(NewsletterLinkEntity::class)->getTableName() . "` DISABLE KEYS");
+    $connection->executeStatement("ALTER TABLE `" . $this->entityManager->getClassMetadata(ScheduledTaskEntity::class)->getTableName() . "` DISABLE KEYS");
+    $connection->executeStatement("ALTER TABLE `" . $this->entityManager->getClassMetadata(ScheduledTaskSubscriberEntity::class)->getTableName() . "` DISABLE KEYS");
+    $connection->executeStatement("ALTER TABLE `" . $this->entityManager->getClassMetadata(SendingQueueEntity::class)->getTableName() . "` DISABLE KEYS");
+    $connection->executeStatement("ALTER TABLE `" . $this->entityManager->getClassMetadata(StatisticsOpenEntity::class)->getTableName() . "` DISABLE KEYS");
+    $connection->executeStatement("ALTER TABLE `" . $this->entityManager->getClassMetadata(StatisticsClickEntity::class)->getTableName() . "` DISABLE KEYS");
+    $connection->executeStatement("SET UNIQUE_CHECKS = 0;");
   }
 
   public function runAfter() {
-    ORM::rawExecute(
+    $connection = $this->entityManager->getConnection();
+
+    $connection->executeStatement(
       "ALTER TABLE `" . $this->entityManager->getClassMetadata(StatisticsClickEntity::class)->getTableName() . "`
       CHANGE `updated_at` `updated_at` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP;"
     );
 
     // Enable keys
     global $wpdb;
     $prefix = $wpdb->prefix;
-    ORM::rawExecute("ALTER TABLE `{$prefix}posts` ENABLE KEYS");
-    ORM::rawExecute("ALTER TABLE `{$prefix}postmeta` DISABLE KEYS");
-    ORM::rawExecute("ALTER TABLE `" . Subscriber::$_table . "` ENABLE KEYS");
-    ORM::rawExecute("ALTER TABLE `" . SubscriberSegment::$_table . "` ENABLE KEYS");
-    ORM::rawExecute("ALTER TABLE `" . $this->entityManager->getClassMetadata(NewsletterLinkEntity::class)->getTableName() . "` ENABLE KEYS");
-    ORM::rawExecute("ALTER TABLE `" . ScheduledTask::$_table . "` ENABLE KEYS");
-    ORM::rawExecute("ALTER TABLE `" . ScheduledTaskSubscriber::$_table . "` ENABLE KEYS");
-    ORM::rawExecute("ALTER TABLE `" . SendingQueue::$_table . "` ENABLE KEYS");
-    ORM::rawExecute("ALTER TABLE `" . $this->entityManager->getClassMetadata(StatisticsOpenEntity::class)->getTableName() . "` ENABLE KEYS");
-    ORM::rawExecute("ALTER TABLE `" . $this->entityManager->getClassMetadata(StatisticsClickEntity::class)->getTableName() . "` ENABLE KEYS");
-    ORM::rawExecute("SET UNIQUE_CHECKS = 1;");
+    $connection->executeStatement("ALTER TABLE `{$prefix}posts` ENABLE KEYS");
```

Comment:
Despite these being old code, I'd suggest moving them to a different method with a meaningful name so that it's easier to know what these similar lines of code do.

Reply:
I agree with the general sentiment here but I got stuck in choosing a name and thus I will suggest, if you agree, that we leave this for the future. The reason why I got stuck is that I'm not even sure if running `DISABLE KEYS` and `ENABLE KEYS` is needed. My understanding is that it only has any effect when MyISAM is being used and I would assume that the majority of the WP installs where this can be executed (mostly dev environments) are using InnoDB. I don't think it is worth investing more time in this in the context of this PR.